README.md

```diff
@@ -12,7 +12,7 @@
 
 ## Live Demo (if available)
 
-[Live Demo Link](file:///C:/Users/athal/Portflio_Setup/index.html)
+[Live Demo Link](https://athaliah8.github.io/Mobile_Portfolio/)
 
 
 
```

Comment:
- [ ] Even though all other reviewers might have overlooked this, kindly consider using the updated version of the [README](https://github.com/microverseinc/readme-template) that was provided by Microverse.